judge/views/contests.py

```diff
@@ -434,6 +434,7 @@ def join_contest(self, request, access_code=None):
                     contest=contest, user=profile, virtual=(-1 if self.is_organizer else 0),
                     real_start=timezone.now(),
                 )
+                event.post('contest_%d' % contest.id, {'type': 'update'})
```

Comment:
Shouldn't you use the new string formatting method?
aka `"{}".format(x)`

Reply:
No. This is not the preferred form on DMOJ code.